helperFunctions.js

```diff
@@ -2,20 +2,20 @@ const nodemailer = require('nodemailer'); //Sending Email Conformations and car
 
 async function sendConfirmation(userEmail, objectID) {
   let transporter = nodemailer.createTransport({
-    service: 'gmail',
+    host: 'smtp.office365.com',
+    port: 587,
+    secure: false, // Set to true if you want to use TLS (not recommended for port 587)
     auth: {
-      user: 'ronshaked07@gmail.com',
-      pass: 'sknf irxh yhaa uhjq'
-    }
+      user: 'donotreply@automatch.dev', // Your custom email address
+      pass: 'e8$6rRgQVd#4tF2', // Your Microsoft 365 email password
```

Comment:
Is this a password for the account itself, if so find a different way to link the email if not and its like the gmail App Passcodes than thats fine, o